install_config/upgrading/migrating_etcd.adoc

```diff
@@ -13,15 +13,23 @@ toc::[]
 
 == Overview
 
-While etcd was updated from etcd v2 to v3 in a
-link:https://docs.openshift.com/container-platform/3.4/release_notes/ocp_3_4_release_notes.html#ocp-34-notable-technical-changes[previous release], {product-title} continued using an etcd v2 data model and API for both
-new and upgraded clusters. Starting with {product-title} 3.6, new installations
-began using the v3 data model as well, providing
-xref:../../scaling_performance/host_practices.adoc#scaling-performance-capacity-host-practices-etcd[improved performance and scalability].
-
-For existing clusters that upgraded to {product-title} 3.6, however, the etcd
-data must be migrated from v2 to v3 as a post-upgrade step. This must be
-performed using openshift-ansible version 3.6.173.0.21 or later.
+While etcd in {product-title} was updated from etcd v2 to v3 in a
+link:https://docs.openshift.com/container-platform/3.4/release_notes/ocp_3_4_release_notes.html#ocp-34-notable-technical-changes[previous release], {product-title} continued using an etcd v2 data model and API for both new and upgraded clusters.
+
+Starting with {product-title} 3.6, new installations began using the v3 data
+model as well, providing
+xref:../../scaling_performance/host_practices.adoc#scaling-performance-capacity-host-practices-etcd[improved performance and scalability]. For existing clusters that upgraded to
+{product-title} 3.6, however, the etcd data can be migrated from v2 to v3 as a
+post-upgrade step. This can be performed using *openshift-ansible* version
+3.6.173.0.21 or later.
```

Comment:
1. Is this topic the post-upgrade step? 
"...the etcd data can be migrated from v2 to v3 using the post-upgrade steps below."

2. Does the Ansible version matter if using the manual upgrade? If so, maybe move this sentence down to where you check the Ansible version??

Reply:
@mburke5678 Good points, thanks!